### PR TITLE
Ignored tests that allocate too much memory

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-file-datalake/pom.xml [hunk 1]
```diff
@@ -104,6 +104,12 @@
       <version>3.2.7</version> <!-- {x-version-update;cglib:cglib-nodep;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+      <version>3.1</version>
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-datalake/pom.xml:110]: Need to add a version tag for this in `external_dependencies.txt`, this is why analyze is failing.

**Comment by rickle-msft** [comment]
\azp run java - storage - tests

**Pull request closed without merging**